### PR TITLE
Pas 72 API Functions High Level

### PR DESCRIPTION
**Pull request by seiyaterada**

### CODE REVIEW

**Review by isaiahdoyle — COMMENTED** [review]
Looks good! I suppose as of now these are just the templates for the actual functions right?

**Comment by seiyaterada** [comment]
@isaiahdoyle Sorry didn't know how to reply to your comment, but ya this ticket is just supposed to be the templates for the routers. Implementing the different routers will be separate tickets :)

**Comment by isaiahdoyle** [comment]
Haha no worries, just wasnt totally sure what the scope ot the ticket was :)

**Review by isaiahdoyle — APPROVED** [review]

File: src/index.ts [hunk 1]
```diff
@@ -1,15 +1,28 @@
...
 
 const port = 5000;
-
 const app = express();
 
+app.use('/', (req, res, next) => {
+	console.log('Successful PassrAPI!');
+	next();
+});
+
```
Comment (BraidenCutforth) [inline src/index.ts:16]: We can probably remove this middleware for now

**Review by BraidenCutforth — APPROVED** [review]

**Review by Samantha-norrie — APPROVED** [review]

**Pull request merged**